appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -411,6 +411,8 @@ private YaVersion() {
   // - FTC_DC_MOTOR_CONTROLLER_COMPONENT_VERSION was incremented to 3.
   // For YOUNG_ANDROID_VERSION 145:
   // - Spelling of "obsfucate" was corrected to obfuscate and BLOCKS_LANGUAGE_VERSION incremented to 20
+  // - FTC_GAMEPAD_COMPONENT_VERSION was incremented to 2.
+  // - FTC_ROBOT_CONTROLLER_COMPONENT_VERSION was incremented to 3.
```

Comment:
Yes, it's safe. I only need to bump the version number if I add/change a component and the master hasn't needed to bump the version number. In this case, these changes were added a month ago, but I hadn't merged in the changes from master, so they were added at 145.